dask/dataframe/tests/test_dataframe.py

```diff
@@ -1612,6 +1612,7 @@ def test_dataframe_picklable():
 
     cloudpickle = pytest.importorskip("cloudpickle")
     cp_dumps = cloudpickle.dumps
+    cp_loads = cloudpickle.loads
```

Comment:
This is needed because of changes in `cloudpickle` itself. Please see Pierre's comment ( https://github.com/dask/dask/issues/6374#issuecomment-655405631 ) for details.

Reply:
Also note that `dask.multiprocessing` [already does the right thing]( https://github.com/dask/dask/blob/7d09b9e505a2686d425b5fa240526583a250e62a/dask/multiprocessing.py#L27 ). So no change was needed there.

Reply:
Alternatively, could we have imported pickle5 as pickle an expect this to work ?  Not saying we should

Reply:
Yep, I think that would work as well. Effectively that is how Python 3.8 avoided this issue during testing.